tests/iid_example.py

```diff
@@ -0,0 +1,67 @@
+import sys
```

Comment:
I think these  example code can be in a separate folder called `examples`. They aren't really meant as tests